numerous/engine/simulation/solvers/numerous_solver/numerous_solver.py

```diff
@@ -3,7 +3,7 @@
 
 from numerous.engine.simulation.solvers.base_solver import BaseSolver
 from .solver_methods import *
-from numba import njit, objmode
+from numba import njit
```

Comment:
It's recommended to have particular order of imports split into blocks by empty line:

1. imports from future
2. built-in modules
3. third-party modules
4. abs local modules
5. rel local modules

For example

```
from __future__ import annotations

import os
from time import sleep

import numpy as np
from numpy import array

import config
from numerous.engine.model.lowering.ast_builder import ASTBuilder

from .solver import numerous_solver
```


Also, it is highly recommended to avoid using wide imports (from ... import *) like

```
from solvers import *
```